File: aroundHubSpringBoot/src/main/java/com/example/aroungHubJavaSpringBoot/aroundHubSpringBoot/data/dto/ProductDto.java
package com.example.aroungHubJavaSpringBoot.aroundHubSpringBoot.data.dto;

import com.example.aroungHubJavaSpringBoot.aroundHubSpringBoot.data.entity.ProductEntity;
import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@ToString
@Builder
public class ProductDto
{
    private String productId;
    private String productName;
    private int productPrice;
    private int productStock;


    public ProductEntity toEntity(){
        return ProductEntity.builder()
                .productId(productId)
                .productName(productName)
                .productPrice(productPrice)
                .productStock(productStock)
                .build();
    }

}

File: aroundHubKotlinSpringBoot/src/main/kotlin/com/example/aroungHubKotlinSpringBoot/aroundHubKotlinSpringBoot/data/dto/ProductDto.kt
package com.example.aroungHubKotlinSpringBoot.aroundHubKotlinSpringBoot.data.dto

import com.example.aroungHubKotlinSpringBoot.aroundHubKotlinSpringBoot.data.entity.ProductEntity

class ProductDto(
    var productId: String? = null,
    var productName: String? = null,
    var productPrice: Int? = null,
    var productStock: Int? = null
) {
    fun toEntity() : ProductEntity{
        return ProductEntity(
            productId = productId,
            productName = productName,
            productPrice = productPrice,
            productStock = productStock
        )
    }
}